qiime2_pipeline.bash

```diff
@@ -0,0 +1,177 @@
+#!/bin/bash
+
+if [ $# -ne 4 ]; then
+	echo "Usage: $0 WORK_DIR MAPPING_FP LANE_NUM PROJECT_NAME"
+	exit 1
+fi
+
+WORK_DIR=$1
+MAPPING_FP=$2
+LANE_NUM=$3
+PROJECT_NAME=$4
+
+## Taxonomy classifier setup. Two classifiers are currently available:
+## classifiers trained on full length and on 515F/806R region of Greengenes 13_8 99% OTUs
+## These can be downloaded from https://data.qiime2.org/2017.9/common/gg-13-8-99-nb-classifier.qza (full length)
+## or https://data.qiime2.org/2017.9/common/gg-13-8-99-515-806-nb-classifier.qza (515F/806R region)
+
+CLASSIFIER_FP="${HOME}/gg-13-8-99-nb-classifier.qza"
+#CLASSIFIER_FP="${HOME}/gg-13-8-99-515-806-nb-classifier.qza" ## used for V4 region
+
+DATA_DIR="${WORK_DIR}/data_files"
+
+PROJECT_DIR="${WORK_DIR}/${PROJECT_NAME}"
+if [ ! -d ${PROJECT_DIR} ]; then
+	mkdir ${PROJECT_DIR}
+fi
+
+FORMATTED_LANE_NUM=$(printf "%03d" ${LANE_NUM})
+FWD="${DATA_DIR}/Undetermined_S0_L${FORMATTED_LANE_NUM}_R1_001.fastq.gz"
+REV="${DATA_DIR}/Undetermined_S0_L${FORMATTED_LANE_NUM}_R2_001.fastq.gz"
+IDX="${DATA_DIR}/Undetermined_S0_L${FORMATTED_LANE_NUM}_I12_001.fastq.gz"
```

Comment:
Suggest incorporating our own script to extract barcodes, rather than doing this manually.